data/morphology/grab_unimorph_data.py

```diff
@@ -1,52 +1,69 @@
 #!/usr/bin/env python
 """Downloads UniMorph morphological paradigms data."""
-
 import json
 import logging
 import os
-import requests
 import time
-
 from typing import Dict
 
 
+""" import requests in separate code block because of third party status """
+import requests
+
+
+start_time = time.time()
+
+
 UNIMORPH_DICT_PATH = "unimorph_languages.json"
 
 
-def download(data_to_grab: Dict[str, str]) -> Dict[str, str]:
-    to_retry = {}
-    os.mkdir("tsv")
-    for language, url in data_to_grab.items():
-        with requests.get(url, stream=True) as response:
-            logging.info("Downloading: %s", language)
+def download():
```

Comment:
Why does this no longer return retries? The loop in `main` doesn't make any sense anymore if so.

Reply:
Please restore the typing on this function from the earlier version too. There's no reason for your PR to touch this function.